server/plugin/plugin.go

```diff
@@ -81,6 +87,48 @@ func (p *MatterpollPlugin) checkServerVersion() error {
 	return nil
 }
 
+// ensureBotAccount checks if the bot account exists and creates him if doesn't
+func (p *MatterpollPlugin) ensureBotAccount() error {
+	botUserID, err := p.Store.Bot().GetID()
+	if err != nil {
+		return errors.Wrap(err, "failed to get bot user from store")
+	}
+	if botUserID == "" {
+		bundlePath, err := p.API.GetBundlePath()
+		if err != nil {
+			return errors.Wrap(err, "failed to get bundle path")
+		}
+
+		profileImage, err := ioutil.ReadFile(filepath.Join(bundlePath, "assets", "logo_dark.png"))
+		if err != nil {
+			return errors.Wrap(err, "failed to read profile image")
+		}
+
+		bot := &model.Bot{
+			Username:    "matterpoll",
+			DisplayName: "Matterpoll",
+			Description: "Some description",
```

Comment:
Should this be localizes? How do we deal with changes of the default server language?

Reply:
Yes, users can see it. The description is shown in the profile pop up. 
![Screenshot from 2019-03-24 19-48-35](https://user-images.githubusercontent.com/16541325/54884166-047a3e00-4e6e-11e9-9e6c-0172cd5e6f4f.png)


:+1: on localizing this. The remaining questions is:

> How do we deal with changes of the default server language?

Reply:
Thanks, I got it.
We can catch changes of server config by `OnConfigurationChage` hooks, and update the description of bot by `Bot.Patch` in that hook func.
Existing poll will not be updated, but it's fine for me.